model/account/accounts.go

```diff
@@ -140,7 +140,7 @@ func GetTriggers(jobsSystem job.JobSystem, db prefixer.Prefixer, accountID strin
 
 	var toDelete []job.Trigger
 	for _, t := range triggers {
-		if t.Infos().WorkerType != "konnector" {
+		if !(t.Infos().WorkerType == "konnector" || t.Infos().WorkerType == "client") {
```

Comment:
nit: I find this kind of condition particularly hard to read as the exclamation mark can be easily missed here.
Would you be okay with using `!= && !=` instead?

Reply:
My brain works differently, I've hard time to understand `!= && !=` but you're in charge here, you do whatever you want :). I'm not maintaining this code, so if it's easier for you to change, let's change it :)

Reply:
OK :). I would prefer that then.

```suggestion
		if t.Infos().WorkerType != "konnector" && t.Infos().WorkerType != "client" {
```

Reply:
This will be taken care of in https://github.com/cozy/cozy-stack/pull/3887 instead.